account_lock_to_date/wizards/account_update_lock_to_date.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@jbeficent , just asking for this. Do we actually need this xml tag for view? Or we don't need anymore as for v12.

Reply:
@kittiu I can see that it's everywhere, also in v12.